packages/cli/src/hooks/analytics/analytics.ts

```diff
@@ -5,53 +5,54 @@ import {
   AuthenticationStatus,
   getAuthenticationStatus,
 } from '../../lib/platform/authenticatedClient';
+import {Identifier} from './identifier';
 
 export interface AnalyticsHook {
   event: Event;
   config: IConfig;
 }
 
-// TODO: CDX-651 track user properties
-// import {Identifier} from './identifier';
-
 // TODO: CDX-656: replace with Production API key on build
 const analyticsAPIKey = '2b06992f1a80d36396ba7297a8daf913';
 
 const hook = async function (options: AnalyticsHook) {
-  const {event} = options;
   if (!(await isLoggedIn())) {
     // TODO: track event with anonymous user
     return;
   }
   const amplitudeClient = configureAmplitudeClient();
-  // TODO: CDX-651: Identify unique users
-  const {organization, analyticsEnabled} = await platformInfoIdentifier();
+  const {organization, analyticsEnabled, authenticatedClient} =
+    await platformInfoIdentifier();
 
   if (!analyticsEnabled) {
     return;
   }
-  const platformClient = await new AuthenticatedClient().getClient({
+
+  // TODO: CDX-676: remove this block to track events from all org types
+  // ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
```

Comment:
Hint, there's `#region` with VSCode that can be useful to mark 'area' of code.
Tho no big deal really here.

Reply:
Good to know... but it's another extension to install